app/src/main/java/com/powerhungers/fomezero/feature/login/presentation/LoginFragment.kt

```diff
@@ -33,11 +33,12 @@ class LoginFragment : Fragment() {
         with(binding) {
             btnEnter.setOnClickListener {
                 viewModel.login(
-                    "luca@fagundes.com","Ab1234567!"
+                    "teste@api.com", "Ab1234567!"
                 )
             }
             txtCadastre.setOnClickListener {
-                findNavController().navigate(LoginFragmentDirections.actionLoginFragmentToRegistrationFragment())
+                findNavController().navigate(
+                    LoginFragmentDirections.actionLoginFragmentToRegistrationFragment())
```

Comment:
```suggestion
                    LoginFragmentDirections.navigateToRegistrationFragment())
```
eu particularmente gosto de simplificar o nome da action de navigation, mas é so uma questao de padrao, qualquer um que vcs definirem é so mante-lo